airflow/providers/amazon/aws/hooks/glue_crawler.py

```diff
@@ -0,0 +1,169 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from time import sleep
+
+from cached_property import cached_property
+
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+
+
+class AwsGlueCrawlerHook(AwsBaseHook):
+    """
+    Interacts with AWS Glue Crawler
+    :param config = Configurations for the AWS Glue crawler
+    :type config = dict
+    """
+
+    def __init__(self, *args, **kwargs):
+        kwargs['client_type'] = 'glue'
+        super().__init__(*args, **kwargs)
+
+    @cached_property
+    def glue_client(self):
+        """:return: AWS Glue client"""
+        return self.get_conn()
+
+    def check_iam_role(self, role_name: str) -> str:
+        """
+        Checks if the input IAM role name is a
+        valid pre-existing role within the caller's AWS account.
+        Is needed because the current Boto3 (<=1.16.46)
+        glue client create_crawler() method misleadingly catches
+        non-existing role as a role trust policy error.
+        :param role_name = IAM role name
```

Comment:
```suggestion
        non-existing role as a role trust policy error.

        :param role_name = IAM role name
```
Please put an empty line between description and params.

Reply:
And check this on the rest of your code, please. Thanks :)